frontend/src/static/js/v_authorship.js

```diff
@@ -319,6 +351,7 @@ window.vAuthorship = {
     this.initiate();
     this.setInfoHash();
   },
+
```

Comment:
maybe the new line is not necessary

Reply:
Hmm, most of the "sections" are currently seperated by newlines though.
e.g. between `watch` and `methods` and `components`.
Not sure if github is displaying this correctly, I wouldn't mind changing it otherwise though

Reply:
Alright, then it is fine